python/pyspark/sql/readwriter.py

```diff
@@ -747,16 +800,25 @@ def _test():
     except py4j.protocol.Py4JError:
         spark = SparkSession(sc)
 
+    seed = int(time() * 1000)
```

Comment:
I have been really busy with work of late, but I will try to sort this out today